docs/platforms/javascript/common/user-feedback/configuration/index.mdx

```diff
@@ -114,6 +114,7 @@ The following options can be configured for the integration in `feedbackIntegrat
 | Key                           | Default                                  | Description                                                                    |
 | ----------------------------- | ---------------------------------------- | ------------------------------------------------------------------------------ |
 | `triggerLabel`                | `"Report a Bug"`                         | The label of the injected button that opens up the feedback form when clicked. |
+| `triggerAriaLabel`            | `triggerLabel` || `"Report a Bug"`       | The aria label of the injected button that opens up the feedback form when clicked. Available starting in SDK v8.20.0. |
```

Comment:
is the `||` intentional?

Reply:
Yes, I want to show that if the `triggerLabel` is empty, then 'Report a Bug' would be used. Open to suggestions on better ways to explain this!

Reply:
ooh i think that syntax might mess up the markdown, i was trying to preview it online since deploy broke, but it looks like this
<img width="743" alt="SCR-20240724-kcid" src="https://github.com/user-attachments/assets/de71362e-800d-47a3-86b2-aeddeeb319ea">

Reply:
tragic